apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2431,23 +2431,24 @@ const uploadFoldersEpic: AppEpic = (action$, state$) =>
       ).pipe(
         switchMap((foldersAndEntities) => {
           const actions: Observable<AnyAction>[] = [];
-          const folders = foldersAndEntities.flatMap((f) => f.folders);
-          const conversations = foldersAndEntities.flatMap((f) => f.entities);
-
-          const publicConversationIds = conversations
-            .filter((conv) => {
-              const rootParentFolder =
-                ConversationsSelectors.selectRootParentFolder(
-                  state$.value,
-                  conv.folderId,
-                );
-
-              return rootParentFolder && rootParentFolder.publishedWithMe;
-            })
-            .map((conv) => conv.id);
-          const { publicVersionGroups, items: publicConversations } =
+          const folders = foldersAndEntities.flatMap(
+            (folder) => folder.folders,
+          );
+          const conversations = foldersAndEntities.flatMap(
+            (items) => items.entities,
+          );
+          const publicConversations = conversations
+            .filter((conv) => isEntityIdPublic(conv))
+            .map((conv) => ({
+              ...conv,
+              updatedAt: conv.lastActivityDate ?? Date.now(),
```

Comment:
Why not `lastActivityDate`?
```suggestion
              lastActivityDate: conv.lastActivityDate ?? Date.now(),
```